apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/DatasetsResource.java

```diff
@@ -340,9 +340,9 @@ public Response createDatasetItems(
 
         String workspaceId = requestContext.get().getWorkspaceId();
 
-        log.info("Creating dataset items batch by datasetId '{}', datasetName '{}', size '{}' on workspaceId '{}'",
-                batch.datasetId(), batch.datasetId(), batch.items().size(), workspaceId);
-        itemService.save(new DatasetItemBatch(batch.datasetName(), batch.datasetId(), items))
+        log.info("Creating dataset items batch by datasetId '{}', datasetName '{}', size '{}' on workspaceId '{}', includeTraceMetadata '{}'",
+                batch.datasetId(), batch.datasetName(), batch.items().size(), workspaceId, batch.includeTraceMetadata());
```

Comment:
The text mentions `datasetId`, but the bind uses `datasetName`. Please fix this inconsistency.